provision/acc_provision/acc_provision.py

```diff
@@ -122,6 +122,7 @@ def config_default():
     default_config = {
         "aci_config": {
             "system_id": None,
+            "refreshtime": 0,  # APIC Subscription refresh interval; default to 0 to handle APIC older versions.
```

Comment:
1. To preserve existing behavior, we actually don't want to set a default here (because 0 is not default on the aci-container-controller side). The user would set to 0 explicitly if he wants the older (3.2x) behavior.
2. Can you call it: apic_refreshtime. I know this goes under the apic section and hence adding "apic_" prefix might seem redundant, but it still helps to more easily correlate with the configuration used in the aci-container-controller.

Reply:
nit pick : refreshtime -> refresh_time

Reply:
I believe not having the '_' was intentional to be consistent with the naming on the aci-container-controller side.

Reply:
Yes, it was intentional to be consistent.

Reply:
> 1. To preserve existing behavior, we actually don't want to set a default here (because 0 is not default on the aci-container-controller side). The user would set to 0 explicitly if he wants the older (3.2x) behavior.

So, you prefer to keep the default as 900 from acc side? I thought defaulting to 0 inturn results into APIC default timeout 90s and gracefully take care of lower versions. Any non-default will be specified by the user anyway. Let me know if you still want me to remove the default 0 setting, will take it out.

Reply:
> So, you prefer to keep the default as 900 from acc side?
Yes.

> I thought defaulting to 0 inturn results into APIC default timeout 90s and gracefully take care of lower versions.
No. And that is because in 3.2.x we cannot send the refreshtime argument in the URI, the APIC will not ignore it, it will throw an error. So we have effectively two defaults on the acc side. One is for 4.x, and another one for 3.2.x. The one for 4.x is actually treated as default (i.e. 900), but if we want the 3.2x behavior, it can be achieved by setting to 0 (which in turn ends up not sending the refreshtime argument, and picking up the default on the APIC side). 

> Let me know if you still want me to remove the default 0 setting, will take it out.
Yes, and thanks! As a side-effect, I believe the existing UTs would not have to be updated either.